src/vue-components/popup.vue

```diff
@@ -481,7 +517,15 @@
       if (this.visible) {
         let time = new Date().getTime()
         this.logger.log(`${time}: component is updated`)
-        this.updatePopupDimensions()
+        let vm = this
+        clearTimeout(this.updateDimensionsTimeout)
+        let timeoutDuration = 0
+        if (this.resizeCount > 1) {
+          timeoutDuration = 1000
+        }
+        this.updateDimensionsTimeout = setTimeout(function () {
```

Comment:
Do we need a timeout here to wait for changes to be rendered by Vue? If so, I'm wondering if we can use a `nextTick` instead as it will probably be more effective (less delay, will run exactly after re-render). If there are problems with `nextTick` then a timeout is OK, I think.

Reply:
This timeout is used not to start after re-render. Updated hook fires each rerender and each changes of the data/props - so updatePopupDimensions is fired much more times in sequence sometimes.
And for this case I put the timeout technique - so it works this way - 
  a) update fires (HTML re-render or data/prop change)
  b) timeout (if delay  was started before and haven't finished) clears
  c) start 1000 delay and then start updatePopupDimensions

So it prevents from very long sequence that could happen here.

Reply:
I think this is going to need some testing.